options.html

```diff
@@ -74,6 +79,43 @@
 
     <div id="contentwrapper">
         <div class="content selected">
+
+            <!-- General -->
```

Comment:
I didn't feel like Context Menu fits in with any other section, so I made a `General` section. Let me know, if you think I should put it elsewhere.